internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/fastclient/FastClientIndividualFeatureConfigurationTest.java

```diff
@@ -62,8 +62,8 @@ public void testServerReadQuota() throws Exception {
     String readQuotaUsageRatio = "." + storeName + "--quota_requested_usage_ratio.Gauge";
     String clientConnectionCountGaugeString = ".server_connection_stats--client_connection_count.Gauge";
     String routerConnectionCountGaugeString = ".server_connection_stats--router_connection_count.Gauge";
-    String clientConnectionCountRateString = ".server_connection_stats--client_connection_count.OccurrenceRate";
-    String routerConnectionCountRateString = ".server_connection_stats--router_connection_count.OccurrenceRate";
+    String clientConnectionCountRateString = ".server_connection_stats--new_client_connection_count.OccurrenceRate";
+    String routerConnectionCountRateString = ".server_connection_stats--new_router_connection_count.OccurrenceRate";
```

Comment:
`new_client_connection_count.OccurrenceRate` sounds weird, so can we rename them into the following?
```
client_connection_request.OccurrenceRate
router_connection_request.OccurrenceRate
```

Reply:
Good call. Updated.